status/data_deliveries_plot.py

```diff
@@ -0,0 +1,58 @@
+import json
+import datetime
+from status.util import dthandler, SafeHandler
+
+
+class DataDeliveryHandler(SafeHandler):
+    """ Handles the api call to proj_staged data
+    Loaded through /api/v1/proj_staged/([^/]*)$
+    """
+    def get(self, search_string=None):
+        staged_files_view = self.application.projects_db.view("project/staged_files_irma")
+        summary_view = self.application.projects_db.view("project/summary")
+        delivered_data = {}
+        if not search_string:
+            last_month = datetime.datetime.now()-datetime.timedelta(days=30)
+            first_term = last_month.isoformat()[:10]
+            second_term = datetime.datetime.now().isoformat()[:10]
+            search_string = "{}--{}".format(first_term, second_term)
+        searches = search_string.split('--')
+        for row in staged_files_view.rows:
+            filesize = 0
+            for key,val in row.value.items():
+                for key2,val2 in val.items():
+                    filesize += int(val2['size_in_bytes'])
```

Comment:
It would be easier to read this part if you changed the names of `key, val` and `key2, val2` to something that describes what they represent.

Reply:
True!